en/clubs/mottagningen/sidebar.md

```diff
@@ -2,7 +2,7 @@
 
 The individuals responsible for the Reception are referred to as Title. If you would like to contact us, you can send an email to [titel@datasektionen.se](mailto:titel@datasektionen.se). If you would like to contact a specific person, their contact information can be found below.
 
-### Det Kongliga Dadderiet 
+### The Royal Daddery
```

Comment:
Ska endast Dadderiet ha "The" men de andra fortfarande "Det"?

Reply:
Fixat!